osm2streets/src/types.rs

```diff
@@ -54,6 +54,9 @@ impl NamePerLanguage {
 pub struct MapConfig {
     /// If true, driving happens on the right side of the road (USA). If false, on the left
     /// (Australia).
+    ///
+    /// Note this is calculated by osm2streets! The value passed in is ignored; don't do any work
```

Comment:
It's a bit weird to let the caller be able to specify this at all. It'd be a bigger refactor to pull this field out elsewhere, and it kind of fits here otherwise. So for now, seems OK